EIPS/eip-7577.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 7577
+title: Versioning Scheme for EIPs
+description: Use a versioning scheme for EIPs based on changes made to their Specification section.
+author: danceratopz (@danceratopz), Ahmad Bitar (@smartprogrammer93)
+discussions-to: https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295
+status: Draft
+type: Meta
```

Comment:
I wonder if this shall be `Informational` or `Meta`. I like the idea and direction. I don't think every EIP needs adopt it